.github/workflows/ci.yml

```
@@ -196,6 +197,11 @@ jobs:
           git config --global user.email 'miri@cron.bot'
       - name: get changes from rustc
         run: ./miri rustc-pull
+      - name: Install rustup-toolchain-install-master
+        if: ${{ steps.cache.outputs.cache-hit != 'true' }}
+        shell: bash
+        run: |
+          cargo install -f rustup-toolchain-install-master
```

```suggestion
        run: cargo install -f rustup-toolchain-install-master
```
we don't have a cache here so the rest doesn't do much (and no idea why we set `shell`...)